### DISCUSSION TITLE
Feature requests: import inventory and bundles

### DISCUSSION THREAD

**Discussion by caleidoscopio** [leihs/leihs#1332] [Ideas]
The possibility to import an inventory list from CSV file would be very helpful.

I would also love the possibility to add multiple inventory items to a bundle, so each item doesn't need to be booked individually.

For example, if I have a Camera body, a lens and a memory card; I'd like to include all into one kit that will be booked simultaneously and those items will show up as unavailable individually.

**Comment by nimaai** [comment]
> The possibility to import an inventory list from CSV file would be very helpful.

not sure if this will be ever provided in future but we are currently working on a API for managing inventory.

> I would also love the possibility to add multiple inventory items to a bundle, so each item doesn't need to be booked individually.

> For example, if I have a Camera body, a lens and a memory card; I'd like to include all into one kit that will be booked simultaneously and those items will show up as unavailable individually.

this is already possible: https://demo.leihs.zhdk.ch/manage/38c315b9-61a9-42f4-8764-80b64e042f26/items/new?forPackage=true&type=item
